src/mappings/Controller/schema.graphql

```diff
@@ -23,7 +23,9 @@ type ControllerScheme @entity {
 	schemeRegistrarParams: SchemeRegistrarParam
 	uGenericSchemeParams: UGenericSchemeParam
 	genericSchemeParams: GenericSchemeParam
-
+	numberOfQueuedProposals: BigInt!
```

Comment:
is that not suppose to be per dao ?

Reply:
yes, indeed it is, good call.

It is probably better to count and store this ont he `gpqueue` object.